src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -774,11 +774,11 @@ internal void UpdateRigiditySlider(float value, int mutationPoints)
             rigiditySlider.Editable = false;
         }
 
-        rigiditySlider.Value = value;
+        rigiditySlider.Value = value * 10;
```

Comment:
It would make more sense if the value was passed as an int to this method. Because OnRigidityChanged sends the value to the editor as is, it would make sense if the value the editor sends here was int as well.